ansible/roles/cyhy_feeds/tasks/main.yml

```diff
@@ -0,0 +1,221 @@
+    headers:
+      Authorization: "token {{ github_oauth_token }}"
+    dest: /var/cyhy/scripts/cyhy-feeds/cyhy-data-extract.py
+    mode: 0755
+
+<<<<<<< HEAD
+- name: Download dmarc extract from GitHub
+  get_url:
+    url: "https://raw.githubusercontent.com/jsf9k/cyhy-feeds/develop/aws_jobs/dmarc.py"
+    headers:
+      Authorization: "token {{ github_oauth_token }}"
+    dest: /var/cyhy/scripts/cyhy-feeds/dmarc.py
+    mode: 0755
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+- name: Grab cyhy extract config file from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: cyhy-data-extract.cfg
+    dest: /tmp/cyhy-data-extract.cfg
+    mode: get
+  become: no
+
+- name: Grab gpg public key from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: ncats_lab_public_gpg.key
+    dest: /tmp/ncats_lab_public_gpg.key
+    mode: get
+  become: no
+
+<<<<<<< HEAD
+- name: Grab cyhy private key from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: ncats_lab_private_gpg.key
+    dest: /tmp/ncats_lab_private_gpg.key
+    mode: get
+  become: no
+
+- name: Grab cyhy johns public key from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: john_public.key
+    dest: /tmp/john_public.key
+    mode: get
+  become: no
+
+- name: Grab gpg trust keys from S3
+  local_action:
+    module: aws_s3
+    bucket: ncats-cyhy-secrets
+    object: gpg_keys.trust
+    dest: /tmp/gpg_keys.trust
+    mode: get
+  become: no
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+#
+# Copy extract config to new instance
+#
+- name: Copy config file for moe data extractor
+  copy:
+    src: /tmp/cyhy-data-extract.cfg
+    dest: /var/cyhy/scripts/cyhy-feeds/cyhy-data-extract.cfg
+    mode: 0644
+
+- name: Copy gpg public key for moe data extract
+  copy:
+    src: /tmp/ncats_lab_public_gpg.key
+    dest: /var/cyhy/scripts/cyhy-feeds/ncats_lab_public_gpg.key
+    mode: 0644
+
+<<<<<<< HEAD
+- name: Copy john's public key for moe data extract
+  copy:
+    src: /tmp/john_public.key
+    dest: /var/cyhy/scripts/cyhy-feeds/john_public.key
+    mode: 0644
+
+- name: Copy gpg private key for moe data extract
+  copy:
+    src: /tmp/ncats_lab_private_gpg.key
+    dest: /var/cyhy/scripts/cyhy-feeds/ncats_lab_private_gpg.key
+    mode: 0644
+
+- name: Copy gpg trust file
+  copy:
+    src: /tmp/gpg_keys.trust
+    dest: /var/cyhy/scripts/cyhy-feeds/gpg_keys.trust
+    mode: 0644
+
+=======
+>>>>>>> 9eef88aa0b349b80d16b734abe790d3a450606af
+- name: Create the /var/cyhy/scripts/cyhy-feeds/cyhy_extracts directory
+  file:
+    path: /var/cyhy/scripts/cyhy-feeds/cyhy_extracts
+    state: directory
+
+<<<<<<< HEAD
+- name: import lab public key
+  shell: "gpg2 --trustdb-name /home/kyle.evers/.gnupg/trustdb.gpg --import /var/cyhy/scripts/cyhy-feeds/ncats_lab_public_gpg.key"
```

Comment:
What is up with the `trustdb.gpg` coming from your local .gnupg directory?  How are the rest of us supposed to run this ansible role?

Reply:
This was not the local, but on the AWS instance. Switching to cyhy user.

Reply:
👍